src/vaults/interfaces/token/IShareToken.sol

```diff
@@ -27,7 +27,6 @@ interface IShareToken is IERC20Metadata, IERC7575Share, IERC1404 {
 
     // --- Errors ---
     error NotAuthorizedOrHook();
-    error FileUnrecognizedParam();
```

Comment:
I needed to remove them due to a diamond inheritance issue of two-parent interfaces with the same error. That for me, would be a "strong" reason in favor of moving errors to implementations